client/src/components/edit/NewExperiences.js

```diff
@@ -0,0 +1,202 @@
+                            <Field
+                                name={`${exp}.startDate`}
+                                component={DatePicker}
+                                style={style.datePicker}
+                                hintText="Start Date"
+                                mode="landscape"
+                            />
+                        </div>
+                        <div style={{marginTop: '6%'}}>
+                            <Field
+                                name={`${exp}.endDate`}
+                                component={DatePicker}
+                                style={style.datePicker}
+                                hintText="End Date"
+                                mode="landscape"
+                            />
+                        </div>
+                        <div>
+                            <Field
+                                name={`${exp}.address`}
+                                component={StyledTextField}
+                                text="Address"
+                                multiLine
+                            />
+                        </div>
+                        <div>
+                            <Field
+                                name={`${exp}.city`}
+                                component={StyledTextField}
+                                text="City"
+                                disabled
+                            />
+                        </div>
+                        <div>
+                            <Field
+                                name={`${exp}.state`}
+                                component={StyledTextField}
+                                text="State"
+                                disabled
+                            />
+                        </div>
+                        <div>
+                            <Field
+                                name={`${exp}.postal`}
+                                component={StyledTextField}
+                                text="Postal Code"
+                                disabled
+                            />
+                        </div>
+                        <div>
+                            <Field
+                                name={`${exp}.country`}
+                                component={StyledTextField}
+                                text="Country"
+                                disabled
+                            />
+                        </div>
+                    </div>
+                    <div>
+                        <FieldArray name={`${exp}.programs`} component={renderPrograms} />
+                    </div>
+                </div>
+            ))}
+        </div>
+    </div>
+)
+
+
+
+const renderPrograms = ({ fields }) => (
+    <div>
+        <div style={{width: "50vw", marginLeft:'15%', marginRight: '15%'}}>
+            <RaisedButton  onClick={() => fields.push()} label="Add Program" style={{marginTop:'3%',
```

Comment:
What happens to empty forms if they press the button too many times? Will there be a way to filter out empty forms or partially completed ones?

Reply:
Should we also prevent them from adding a program (clicking the button) until all required fields of the organization information is filled out?